pubspec.yaml

```diff
@@ -64,30 +64,35 @@ flutter:
   # the material Icons class.
   uses-material-design: true
   assets:
-    - assets/icons/app_icon.png
-    - assets/raw/community_mock.webp
-    - assets/raw/profit_mock.webp
-    - assets/raw/placeholder.webp
-
-    - assets/icons/community_selected.png
-    - assets/icons/community.png
-    - assets/icons/discover_selected.png
-    - assets/icons/discover.png
-    - assets/icons/my_plot_selected.png
-    - assets/icons/my_plot.png
-    - assets/icons/profit_loss_selected.png
-    - assets/icons/profit_loss.png
-    - assets/icons/profit_add.png
-
-    - assets/privacy_policy.html
-    - assets/icons/flag_kenya.png
-    - assets/icons/flag_usa.png
-    - assets/icons/detail_icon_sale.png
-    - assets/icons/detail_icon_cost.png
-    - assets/icons/radio_button_default.png
-    - assets/icons/radio_button_active.png
-
-
+  - assets/icons/app_icon.png
```

Comment:
be careful with tabbing on yaml files it can cause a crash. Tab it like the "removed code" 👆👆👆👆

Reply:
Done